### PR TITLE
docs: Fix reference to essentials configuration 

### PR DESCRIPTION
**Pull request by josh-degraw**
Issue:

## What I did
The documentation for the essentials addon was showing the configuration example for the actions addon. I updated it to actually show adding the essentials addon.

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: docs/snippets/common/storybook-main-register-essentials-addon.js.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+```js
+// .storybook/main.js
+
+module.exports = {
+  addons: ['@storybook/addon-essentials'],
+};
```
Comment (jonniebigodes) [inline docs/snippets/common/storybook-main-register-essentials-addon.js.mdx:6]: @josh-degraw if you wouldn't mind updating this snippet and adding the stories field and the other addon that is installed as well. Turning this into:
```js
module.exports = {
  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
};

```
Reasoning behind this is to keep it consistent with the other examples used in our docs.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@josh-degraw looks good, one small item to address. Let me know once you've addressed and I'll be more than glad to merge it. Thanks for helping us out and improving our documentation, we appreciate it.

Hope you have a great weekend!

Stay safe

File: docs/snippets/common/storybook-main-register-essentials-addon.js.mdx (hunk 1 above)
Comment (josh-degraw) [reply docs/snippets/common/storybook-main-register-essentials-addon.js.mdx]: Done

**Comment by jonniebigodes** [comment]
@josh-degraw thanks for taking care of this so promptly, appreciate it 🙏 ! All good on my end, going to merge this!

Hope you have a great day!

Stay safe

**Pull request merged**